### PR TITLE
Changed schema title for the same names

### PR DESCRIPTION
**Pull request by berenda-andrey**
This is PR related with #14

### CODE REVIEW

File: requirements/test.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # requirements for running the tests via pytest
 freezegun==0.3.11
 mock==2.0.0
-pytest>=3.6
+pytest<5.0.0>=3.6
```
Comment (berenda-andrey) [inline requirements/test.txt]: new version of pytest(5.0.0) do not run with pytest-django

Comment (mofr) [reply requirements/test.txt]: Add this comment to the file

File: tests/schema/test_get_schema_title.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+
+@dataclasses.dataclass()
+class DTO:
+    id: int
+
+
+def test_get_schema_title():
+    @winter.core.component_method
+    def method(dto: DTO, second_dto: DTO):
```
Comment (mofr) [inline tests/schema/test_get_schema_title.py]: It should return the same type. Only different types should return different schemas

Comment (mofr) [reply tests/schema/test_get_schema_title.py]: So you have to declare DTO in different files

**Pull request merged**